pkg/azuredx/adxauth/aad_client.go

```diff
@@ -16,7 +16,7 @@ import (
 
 // Abstraction over confidential.Client from MSAL for Go
 type aadClient interface {
-	AcquireTokenOnBehalfOf(ctx context.Context, userAssertion string, scopes []string) (confidential.AuthResult, error)
+	AcquireTokenOnBehalfOf(ctx context.Context, userAssertion string, scopes []string, options ...confidential.AcquireOnBehalfOfOption) (confidential.AuthResult, error)
```

Comment:
Is this the breaking change @kostrse?

Reply:
Not in Grafana Azure SDK.   Grafana Azure SDK references a newer version of Azure SDK as a dependency, which has some contracts changed. So when updating Grafana Azure SDK,   Azure SDK as intermediate dependency needs to be updated as well.